.github/workflows/build.yml

```diff
@@ -28,9 +28,13 @@ jobs:
 
 #----------------------------------------------------------------------------
       - name: Linux Get dependencies
-        if: matrix.platform == 'ubuntu-18.04'
+        if: matrix.platform == 'ubuntu-18.04' && matrix.os_name != 'linux_openresty' && matrix.os_name != 'linux_tengine'
         run: sudo apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl etcd
 
+      - name: Linux Get dependencies
+        if: matrix.platform == 'ubuntu-18.04' && (matrix.os_name == 'linux_openresty' || matrix.os_name == 'linux_tengine')
+        run: sudo apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl etcd-client
```

Comment:
I don't think we need to install `etcd-client`

Reply:
This test example uses the command `etcdctl`.
https://github.com/apache/apisix/blob/master/t/core/etcd-auth.t#L29